debian/changelog

```diff
@@ -1,3 +1,10 @@
+pgcron (1.6.0.citus-1) stable; urgency=low
+
+  * Official 1.6.0 release of PG Cron
+  * Adds support for PostgreSQL 16
+
```

Comment:
We did not use to have an empty line here for earlier versions.

Reply:
All of them are standard. Do I miss sth?

Reply:
It appears that my browser was rendering a strange editor view. Your changes are totally fine in this part of the file.